### PR TITLE
Upgrade ember + add ember sparkles

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
For my own reference:

```
cd ./frameworks/keyed/ember
yarn && yarn build-prod
```

```
cd ./frameworks/keyed/ember-sparkles
yarn && yarn build-prod
```

```
yarn && yarn start
```

```
cd ./webdriver-ts
yarn && yarn build-prod
yarn selenium --count 3 --framework ember
yarn selenium --count 3 --framework ember-sparkles
```

```
cd ./webdriver-ts-results
yarn && yarn build-prod
# => error -- react has type errors. :-\
rm -rf node_modules
npm install
# (yarn aggressively upgrades things within range of semverrules)
npm run build-prod
```

```
cd ./webdriver-ts
yarn results
```

visit http://localhost:8080/webdriver-ts-results/table.html

my results!:

Not sure I set things up correctly, cause I was under the impression that sparkles would be faster. Hmm

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
re-added WIP to add babel 7

**Comment by NullVoxPopuli** [comment]
ready for review!

**Comment by krausest** [comment]
Results are up now. Here's a (cross browser versions, OS versions) comparison with the older versions:
bildschirmfoto 2018-10-04 um 21 28 35
Does it meet your expectations?

File: frameworks/keyed/ember-sparkles/app/templates/components/row.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<tr class={{@rowClass}}>
+  <td class="col-md-1">{{@item.id}}</td>
+
+  <td class="col-md-4">
+    <a style='cursor: pointer' onclick={{action @onSelect}}>
```
Comment (cibernox) [inline frameworks/keyed/ember-sparkles/app/templates/components/row.hbs]: You don't need to wrap `onSelect` on an action, it is already an action.

File: frameworks/keyed/ember-sparkles/app/templates/components/row.hbs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      {{@item.label}}
+    </a>
+  </td>
+
+  <td class="col-md-1">
+    <a onclick={{action @onRemove}}>
```
Comment (cibernox) [inline frameworks/keyed/ember-sparkles/app/templates/components/row.hbs]: Same

**Pull request merged**